### PR TITLE
Feature/token

### PR DESCRIPTION
**Pull request by muckmuck96**

### CODE REVIEW

File: src/ormconfig.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { ConnectionOptions } from 'typeorm';
 // Check typeORM documentation for more information.
 const config: ConnectionOptions = {
   type: 'mysql',
-  host: process.env.DATABASE_HOST || 'localhost',
+  host: process.env.DATABASE_HOST || '192.168.99.100',
```
Comment (koanbockFH) [inline src/ormconfig.ts]: NOPE

**Review by koanbockFH — CHANGES_REQUESTED** [review]
Have to debug/test it still but code is ok with 1 exception

**Comment by koanbockFH** [comment]
So test results:

if i log out, it works but the response is 403 on logout - which should be 200.
All other requests (get Profile usw. results in 403 - Forbidden resource)
image

File: src/ormconfig.ts (hunk 1 above)
Comment (muckmuck96) [reply src/ormconfig.ts]: Ups

**Review by koanbockFH — APPROVED** [review]
LGTM

**Pull request merged**